domain/noticecomment/src/main/java/com/ku_stacks/ku_ring/domain/noticecomment/repository/NoticeCommentRepository.kt

```diff
@@ -0,0 +1,26 @@
+package com.ku_stacks.ku_ring.domain.noticecomment.repository
+
+import com.ku_stacks.ku_ring.domain.NoticeComment
+
+interface NoticeCommentRepository {
+    suspend fun createComment(
+        noticeId: Int,
+        parentCommentId: Int?,
+        content: String,
+    ): Result<Boolean>
+
+    suspend fun deleteComment(
+        noticeId: Int,
+        commentId: Int,
+    ): Result<Boolean>
+
+    suspend fun getComment(
+        noticeId: Int,
+        cursor: Int?,
+        size: Int = PAGE_SIZE,
+    ): Result<Pair<List<NoticeComment>, Boolean>>
+
+    companion object {
+        const val PAGE_SIZE = 20
+    }
```

Comment:
```suggestion
    companion object {
        const val PAGE_UNIT = 20
    }
```

페이징의 단위이다 보니 `UNIT`이 좀 더 적절할 것 같네요

Reply:
안드로이드 `PagingConfig`에서도 `pageSize`라는 변수명을 쓰고 있어서 size가 나을 거 같은데 어떠신지..?

Reply:
오 그런 이유가....👍 굿입니다